packages/risc0-rust/default.nix

```diff
@@ -17,8 +17,13 @@ stdenv.mkDerivation rec {
 
   installPhase = ''
     runHook preInstall
-    mkdir -p $out
-    cp -r ./* $out/
+
+    mkdir -p "$out"
+    cp -r ./* "$out/"
+
+    mkdir -p "$out/r0.${version}-risc0-rust-x86_64-unknown-linux-gnu"
```

Comment:
Leave a comment along the lines of "# risc0 requires specific dir name".

Reply:
Done, thank you for the suggestion